### ISSUE TITLE
Error always appears on logs even when plan-all executes successfully

### ISSUE THREAD

**Issue by michelzanini** [gruntwork-io/terragrunt#1220]
I just created a empty `main.tf` and `terragrunt.hcl` to confirm it's not just on my project that is happening.

When you run `terragrunt plan-all` this message is always printed, one for each sub-folder:

```
[terragrunt] 2020/06/17 12:59:17 Error with plan: [terragrunt] [/Users/user/git/project/environments/sandbox] 2020/06/17 12:58:41 Running command: terraform --version
```

`plan-all` still finishes with success and the command works, but the logs always display `Error with plan` on command `terraform --version`.

I am using latest Terraform and Terragrunt versions (`0.12.26` and `0.23.27`).

If this is a bug? Can we fix it so it does not display errors?

Thanks.

**Comment by michelzanini** [comment]
`apply-all` does not seem to have this issue.

**Comment by brikis98** [comment]
Good point! Probably a bug in [this logic](https://github.com/gruntwork-io/terragrunt/blob/37832d50c576d4b4e120e17baf13bdec516110b3/configstack/stack.go#L74-L76). Anyone up for a PR to fix it?

**Comment by michelzanini** [comment]
This issue can now be closed as it has been fixed.
